src/sentry/integrations/slack/tasks.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import absolute_import
+
+import six
+
+from django.conf import settings
+
+from sentry import http
+from sentry.utils import json
+from sentry.tasks.base import instrumented_task
+from sentry.models import Integration, Project, Rule
+
+from sentry.utils.redis import redis_clusters
+from sentry.mediators import project_rules
+
+MEMBER_PREFIX = "@"
```

Comment:
Feels like these constants belong to a higher-level module

Reply:
@MeredithAnya Seems like we already have this defined in the slack util. Can we just import from there?